src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -327,6 +342,12 @@ public class SuppressWithPlainTextCommentFilter extends AutomaticBean implements
     /** Default check format to suppress. By default, the filter suppress all checks. */
     private static final String DEFAULT_CHECK_FORMAT = ".*";
 
+    /** Default regex for lines that should be suppressed. */
+    private static final String DEFAULT_INFLUENCE_FORMAT = "0";
+
+    /** Specify comment pattern to trigger filter to begin suppression. */
+    private Pattern commentFormat = Pattern.compile(DEFAULT_COMMENT_FORMAT);
```

Comment:
I was thinking naming this `defaultCommentFormat` since in the enum down below I chose `DEFAULT`?